docs/index.md

```diff
@@ -60,6 +60,7 @@ ROCm documentation is organized into the following categories:
 * [AMD Instinct MI300X tuning guides](./how-to/tuning-guides/mi300x/index.rst)
   * [System tuning](./how-to/tuning-guides/mi300x/system.rst)
   * [Workload tuning](./how-to/tuning-guides/mi300x/workload.rst)
+* {doc}`GPU cluster networking guides<gpu-cluster-networking:index>`
```

Comment:
This isn't creating a link for some reason, it's saying "doc not found"in the log. Is that because the repo is still private?

Reply:
Look like the issues is that this repo needs to be added to [projects.yaml](https://github.com/ROCm/rocm-docs-core/blob/develop/src/rocm_docs/data/projects.yaml) in rocm-docs-core. @samjwu should I submit a separate PR for this or is there another process that should be followed?

Reply:
Is this intended for 6.2.x? @mbenavidez-amd

Reply:
@peterjunpark I meant for it to target the release that was current at the time I submitted the PR (6.2.0), but didn't know 6.2.1 was going to release a day or two later. Since the content is up, I'd prefer the change be added out-of-cycle rather than waiting for the next release in October.

Reply:
> Look like the issue is that the gpu-cluster-networking repo needs to be added to [projects.yaml](https://github.com/ROCm/rocm-docs-core/blob/develop/src/rocm_docs/data/projects.yaml) in rocm-docs-core. @samjwu should I submit a separate PR for this or is there another process that should be followed?

I'd recommend using an absolute URL for now, and submitting a PR to use the intersphinx link syntax once we've added this to the projects.yaml in rocm-docs-core. 
Just so there is a working link until the next rocm-docs-core package with the intersphinx links are published.

Reply:
I've updated to an absolute link, ready to move forward if there are no other issues.